tests/integration/test_rids/test_rids.py

```diff
@@ -103,6 +157,35 @@ def set_recv_counter_flush(new_recv_counter):
 
 
 def test_rids(get_configuration, configure_environment, restart_service):
+    '''
+    description: Check that RIDS is opened and closed as expected. To do this, it creates injectors(agents and senders)
+                 to be able to communicate with the manager. Then, it stops the agents' listening and checks if RIDS is
+                 closed(when it`s needed).
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - get_configuration:
+            type: fixture
+            brief: Get configuration from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing
+        - restart_service:
+            type: fixture
+            brief: Method to restart the service.
+
+    assertions:
+        - Verify that every agent rid is open
+        - Verify that every agent rid is closed
```

Comment:
I would add a dot (.) at the end of each assertions phrase.
```suggestion
    assertions:
        - Verify that every agent rid is open.
        - Verify that every agent rid is closed.
```

Reply:
https://github.com/wazuh/wazuh-qa/commit/93ae950e1f55e84b0ad12f9ef4dc99cd7e36b493 solves this